docs/user/_index.md

```diff
@@ -1,87 +1,89 @@
 +++
-title = "Grafana Agent documentation"
+title = "Grafana Agent"
 weight = 1
 +++
 
 # Grafana Agent
 
-Grafana Agent is a telemetry collector for sending metrics, logs,
-and trace data to the opinionated Grafana observability stack. It works best
-with:
+Grafana Agent collects and forwards telemetry data to open source deployments of the LGTM (Loki, Grafana, Tempo, Mimir) Stack, Grafana Cloud, or Grafana Enterprise, where your data can then be analyzed. You can install Grafana Agent on Kubernetes and Docker, or as a system process for Linux, macOS, and Windows machines.  
 
-* [Grafana Cloud](https://grafana.com/products/cloud/)
-* [Grafana Enterprise Stack](https://grafana.com/products/enterprise/)
-* OSS deployments of [Grafana Loki](https://grafana.com/oss/loki/), [Prometheus](https://prometheus.io/), [Cortex](https://cortexmetrics.io/), and [Grafana Tempo](https://grafana.com/oss/tempo/)
+The Grafana Agent is open source and its source code is available on GitHub at (https://github.com/grafana/agent).
 
-The Agent supports collecting telemetry data by utilizing the same battle-tested
-code from the official platforms. It uses Prometheus for metrics collection,
-Grafana Loki for log collection, and [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) for trace
-collection.
+Grafana Agent is for engineers, operators, or administrators who want to collect and forward telemetry data for analysis and on-call alerting. Those operating Grafana Agent must install and configure Grafana Agent to properly collect telemetry data and monitor the health of running agents.
 
-Grafana Agent uses less memory on average than Prometheus – by doing less (only focusing on `remote_write`-related functionality).
+## Grafana Agent features
 
-Grafana Agent allows for deploying multiple instances of the Agent in a cluster and only scraping metrics from targets that running at the same host. This allows distributing memory requirements across the cluster rather than pressurizing a single node.
+There are other ways of sending metrics, logs and traces to the Grafana Stack or Grafana Cloud or, but there are a few advantages of using the Grafana Agent. These features are outlined below.
+
+* Provides a one-stop solution for collecting metrics, logs, and traces
+* Collects out-of-the-box telemetry from popular projects like MySQL through integrations
+* Works seamlessly with the Grafana Stack. Alternatively, metrics can be sent to any Prometheus-compatible endpoint, and traces can be sent to any OTLP-compatible endpoint.  
+* Offers new solutions to help scale metrics collection like host_filtering and sharding 
+* Provides the Grafana Agent Operator, which enables individual teams to manage their configurations through PodMonitors, ServiceMonitors, and Probes.
 
 ## Metrics
 
-Unlike Prometheus, the Grafana Agent is _just_ targeting `remote_write`,
+Unlike Prometheus, the Grafana Agent targets `remote_write`,
```

Comment:
I fear readers might misinterpret this as saying Prometheus doesn't support `remote_write`. 

Should we try to rephrase to something like this?

> Grafana Agent focuses metrics support around Prometheus' `remote_write` protocol.

Reply:
done